runtime/crab-parachain/src/pallets/remote_governance.rs

```diff
@@ -0,0 +1,46 @@
+// --- paritytech ---
+use frame_support::traits::{EnsureOrigin, Get};
+use frame_system::RawOrigin;
+// --- darwinia-network ---
+use crate::*;
+use dp_common_runtime::remote_governance::Config;
+
+pub struct EnsureSpecific;
+impl<O: Into<Result<RawOrigin<AccountId>, O>> + From<RawOrigin<AccountId>>> EnsureOrigin<O>
+	for EnsureSpecific
+{
+	type Success = AccountId;
+
+	fn try_origin(o: O) -> Result<Self::Success, O> {
+		o.into().and_then(|o| match o {
+			RawOrigin::Signed(who)
+				if who
+					== array_bytes::hex_into_unchecked(
+						"0x129d025b24257aabdefac93d00419f06a38e3a5e2314dd6866b16e8f205ce074",
+					) =>
+				Ok(who),
+			r => Err(O::from(r)),
+		})
+	}
+}
+
+pub struct CrabBestFinalized;
+impl Get<Hash> for CrabBestFinalized {
+	fn get() -> Hash {
+		<pallet_bridge_grandpa::BestFinalized<Runtime, WithCrabGrandpa>>::get()
+	}
+}
+
+frame_support::parameter_types! {
+	pub const CheckInterval: BlockNumber = 2 * HOURS;
```

Comment:
```suggestion
	pub const CheckInterval: BlockNumber = 4 * HOURS;
```

Reply:
I forgot that 1 epoch of crab is 6 hours,

My suggestion is 6 hours or longer, 1 day LGTM.